module/VuFind/src/VuFind/ILS/Driver/Evergreen.php

```diff
@@ -289,12 +289,8 @@ public function getHolding($id, ?array $patron = null, array $options = [])
                     break;
             }
 
-            if ($row['due_year']) {
-                $due_date = $row['due_year'] . '-' . $row['due_month'] . '-' .
-                            $row['due_day'];
-            } else {
-                $due_date = '';
-            }
+            $due_date = $row['due_year'] ? $row['due_year'] . '-' . $row['due_month'] . '-' .
+                        $row['due_day'] : '';
```

Comment:
We can unwrap this now that line length is higher, to eliminate the excessive indentation on the second line:

```suggestion
            $due_date = $row['due_year'] ? $row['due_year'] . '-' . $row['due_month'] . '-' . $row['due_day'] : '';
```